dotCMS/src/main/java/com/dotcms/rest/api/v1/folder/FolderHelper.java

```diff
@@ -118,4 +118,25 @@ public FolderResults createFolders(List<String> paths, String siteName, User use
         return new FolderResults(savedFolders, errors);
     }
 
+    /**
+     * Return the folder for the passes in URI.
+     *
+     * @param siteName Folder to lookup the URI on
+     * @param user
+     * @param uri The URI (Path) to get the folder for. If you pass / will return the SystemFolder
+     * @return List of assets that the given user has permissions to see under a URI on a Host. (Currently this only returns Folder and File assets)
+     * @throws com.dotmarketing.exception.DotDataException if one is thrown when the sites are search
+     * @throws com.dotmarketing.exception.DotSecurityException if one is thrown when the sites are search
+     */
+    public Folder loadFolderByURI(String siteName, User user, String uri) throws DotDataException, DotSecurityException {
+        Host host = hostAPI.findByName(siteName,user,true);
+        Folder ret = null;
+        if(uri.equals("/")){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "/" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)